app/Helpers/Helper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Helpers;
+
+use Exception;
+
+class Helper
+{
+	/**
+	 * @throws Exception
+	 */
+	public static function secondsToHumanReadable ( int $duration ) : string
+	{
+		if ( $duration < 0 )
+		{
+			throw new Exception( 'Time cannot be lower than zero' );
+		}
+
+		if ( $duration < 60 ) {
+			$result = 'The time is lower than a minute';
+		} else {
+			$days    = floor( $duration / 86400 );
+			$hours   = floor( ( $duration - $days * 86400 ) / 3600 );
+			$minutes = floor( ( $duration / 60 ) % 60 );
+
+			$days    = ( $days == 0 ? '' : $days . ( $days > 1 ? 'days ' : 'd ' ) );
+			$hours   = ( $hours == 0 ? '' : $hours . ( $hours > 1 ? 'hrs ' : 'hr ' ) );
+			$minutes = ( $minutes == 0 ? '' : $minutes . ( $minutes > 1 ? 'mins' : 'min' ) );
+
+			if ( $days != 0 )
```

Comment:
PHP-de == ve === ferqi var: [https://www.educative.io/answers/difference-between-the-and-operators-in-php](https://www.educative.io/answers/difference-between-the-and-operators-in-php)

Senin if-de eslinde bug ola bilecek shert var, sadece PHP bir az qeribe dildi deye duzgun ishleyib. Sen if-den yuxarida yazmisan ki, eger `$days == 0` olarsa, o zaman `$days = ''`, yeni artiq $days `int` yox, `string` olacaq. Ve ashagida ise if shertinde `$days != 0` yazmisan. Normalda `$days !== 0` yazmali idin ve o halda da `'' !== 0` olacagi ucun shert odenmish olacaqdi. Iki cur duzelde bilersen, ya $days-i `''` evezine `0` ele. Ya da if icinde yoxlamani bele yaz `if ( ! empty( $days ) )`